ci-operator/step-registry/openshift-extended/test/longduration/openshift-extended-test-longduration-commands.sh

```
@@ -4,6 +4,36 @@ set -o nounset
+DEBUG_WAIT_TIMEOUT="${DEBUG_WAIT_TIMEOUT:-10800}"
+echo "=========================================="
+echo "DEBUG WAIT ACTIVE - Cluster ready for debugging"
+echo "=========================================="
+echo "Timeout: ${DEBUG_WAIT_TIMEOUT} seconds"
+echo "To continue tests, create the signal file:"
+echo "  oc rsh <test-pod>"
+echo "  touch /tmp/continue"
+echo "=========================================="
+
+elapsed=0
+while [ ! -f "/tmp/continue" ] && [ $elapsed -lt $DEBUG_WAIT_TIMEOUT ]
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)